src/WalletInit/WalletInitNavigator.tsx

```diff
@@ -116,6 +116,8 @@ export const WalletInitNavigator = () => {
   )
 }
 
+const ConnectNanoXScreenWrapper = () => <ConnectNanoXScreen />
```

Comment:
does this do anything?

Reply:
yap, `ConnecteNanoXScreen` has props, ts is complaining, so it was needed to wrap.